src/components/Tabs/Tab/Tab.tsx

```diff
@@ -29,19 +32,25 @@ export const Tab: FC<TabProps> = ({
         { [styles.inverse]: light },
     ]);
 
-    const getIcon = (): JSX.Element => (
-        <Icon path={icon} className={styles.icon} />
-    );
+    const getIcon = (): JSX.Element =>
+        iconExists && <Icon path={icon} className={styles.icon} />;
 
-    const getLabel = (): JSX.Element => (
-        <span className={styles.label}>{label}</span>
-    );
+    const getLabel = (): JSX.Element =>
+        labelExists && <span className={styles.label}>{label}</span>;
 
-    const getTabIndicator = (): JSX.Element => (
-        <Flipped flipId="tabIndicator">
-            <div className={styles.tabIndicator} />
-        </Flipped>
-    );
+    const getTabIndicator = (): JSX.Element =>
+        isActive && (
+            <Flipped flipId="tabIndicator">
+                <div className={styles.tabIndicator} />
+            </Flipped>
+        );
+
+    const getBadge = (): JSX.Element =>
+        !!badgeContent && (
+            <Badge active={isActive} className={styles.badge}>
+                {badgeContent}
+            </Badge>
+        );
 
     return (
         <button
```

Comment:
Please add aria props to enable devs to pass in any relevant information, like the count of the badge.